src/box/lua/error.cc

```diff
@@ -38,6 +38,7 @@ extern "C" {
 
 #include <fiber.h>
 #include <errinj.h>
+#include <tweaks.h>
```

Comment:
Nit: It seems, it is from an old version of the patch and it is not needed anymore.

Reply:
Fixed.